net-irc/inspircd/inspircd-3.17.1.ebuild

```
@@ -0,0 +1,113 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd toolchain-funcs
+
+DESCRIPTION="Inspire IRCd - The Stable, High-Performance Modular IRCd"
+HOMEPAGE="https://www.inspircd.org/"
+SRC_URI="
+	https://github.com/inspircd/inspircd/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/clinew/gentoo-distfiles/raw/master/inspircd-${PV}-fix-build-paths.patch.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+IUSE="argon2 debug gnutls ldap maxminddb mbedtls mysql pcre pcre2 postgres re2 regex-posix regex-stdlib sqlite ssl sslrehashsignal tre"
+
+RDEPEND="
+	acct-group/inspircd
+	acct-user/inspircd
+	dev-lang/perl
+	argon2? ( app-crypt/argon2 )
+	gnutls? ( net-libs/gnutls:= dev-libs/libgcrypt:0 )
+	ldap? ( net-nds/openldap:= )
+	maxminddb? ( dev-libs/libmaxminddb:= )
+	mbedtls? ( net-libs/mbedtls:= )
+	mysql? ( dev-db/mysql-connector-c:= )
+	pcre? ( dev-libs/libpcre )
+	pcre2? ( dev-libs/libpcre2 )
+	postgres? ( dev-db/postgresql:= )
+	re2? ( dev-libs/re2:= )
+	sqlite? ( >=dev-db/sqlite-3.0 )
+	ssl? ( dev-libs/openssl:= )
+	tre? ( dev-libs/tre )"
+DEPEND="${RDEPEND}"
+
+DOCS=( docs/. .configure/apparmor )
+PATCHES=( "${WORKDIR}"/${P}-fix-build-paths.patch )
+
+src_configure() {
+	local extras=""
+
+	use argon2 && extras+="argon2,"
+	use gnutls && extras+="ssl_gnutls,"
+	use ldap && extras+="ldap,"
+	use maxminddb && extras+="geo_maxmind,"
+	use mbedtls && extras+="ssl_mbedtls,"
+	use mysql && extras+="mysql,"
+	use pcre && extras+="regex_pcre,"
+	use pcre2 && extras+="regex_pcre2,"
+	use postgres && extras+="pgsql,"
+	use re2 && extras+="regex_re2,"
+	use regex-posix && extras+="regex_posix,"
+	use regex-stdlib && extras+="regex_stdlib,"
+	use sqlite && extras+="sqlite3,"
+	use ssl && extras+="ssl_openssl,"
+	use sslrehashsignal && extras+="sslrehashsignal,"
+	use tre && extras+="regex_tre,"
+
+	# The first configuration run enables certain "extra" InspIRCd
+	# modules, the second run generates the actual makefile.
+	if [[ -n "${extras}" ]]; then
```

```suggestion
	if [[ -n ${extras} ]]; then
```
You don't need the quotes inside `[[` `]]`

---

Done!